### PR TITLE
Fix incorrect behavior when hmi send GetInteriorVehicleData

### PR DESCRIPTION
**Pull request by okozlovlux**
Removed OnApplicationRegistered notification from RegisterAppInterfaceResponse method Run().

Related to [SDLOPEN-698](https://adc.luxoft.com/jira/browse/SDLOPEN-698)

### CODE REVIEW

**Comment by okozlovlux** [comment]
@LevchenkoS @istoilovgithub @dtrunov @LuxoftAKutsan @indlu please review

File: src/components/application_manager/src/commands/mobile/register_app_interface_response.cc [hunk 1]
```diff
@@ -78,11 +78,6 @@ void RegisterAppInterfaceResponse::Run() {
 
   SetHeartBeatTimeout(connection_key(), application->policy_app_id());
 
-  // Default HMI level should be set before any permissions validation, since
-  // it
-  // relies on HMI level.
-  application_manager_.OnApplicationRegistered(application);
```
Comment (LuxoftAKutsan) [inline src/components/application_manager/src/commands/mobile/register_app_interface_response.cc]: @okozlovlux in develop this code is in response. Please do the same

Comment (okozlovlux) [inline src/components/application_manager/src/commands/mobile/register_app_interface_response.cc]: @LuxoftAKutsan , this code is another branches, but not in develop. see [here](https://github.com/smartdevicelink/sdl_core/blob/develop/src/components/application_manager/src/commands/mobile/register_app_interface_response.cc)

**Comment by okozlovlux** [comment]
Rebuild required

**Comment by okozlovlux** [comment]
Rebuild required

**Pull request merged**